src/core/SpriteAnimator.tsx

```diff
@@ -102,23 +206,23 @@ export const SpriteAnimator: React.FC<SpriteAnimatorProps> = /* @__PURE__ */ Rea
     },
     fref
   ) => {
-    const ref = React.useRef<any>()
-    const spriteData = React.useRef<any>(null)
-    const matRef = React.useRef<any>()
-    const spriteRef = React.useRef<any>()
+    const ref = React.useRef<THREE.Group>(new THREE.Group())
```

Comment:
```suggestion
    const ref = React.useRef(new THREE.Group())
```
since you pass `new THREE.Group()` TS will infer the returned ref type

Reply:
I removed these where applicable